### ISSUE TITLE
ESP32-S3 does not have a pin 34 (Setup70d_ILI9488_S3_Parallel.h)

### ISSUE THREAD

**Issue by MathieuFennema** [Bodmer/TFT_eSPI#2515]
Hello there,

I'm using your beautiful library with ease. and recently switched to an ESP32-S3 with an ILI9488 8-bit parallel interface.
So I stumbled unto the Setup70d_ILI9488_S3_Parallel.h configuration file. 
I noticed that the TFT_RST definition was set to pin 34, however, there is no pin 34 on my [devkit](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1-v1.0.html).

So I set it to pin 17, and it worked just fine.
Thank you for putting the effort in this library, you're supporting a lot of hobbyists out there.

Kind regards,
Mathieu

**Comment by Bodmer** [comment]
Yes, you have to adapt the e ample setups according to the processor board you have and the GPIO made available.
